packages/governance/src/types.js

```diff
@@ -205,6 +205,12 @@
  * @returns {VoteCounterFacets}
  */
 
+/**
+ * @callback LooksLikeApiInvocation
```

Comment:
why name this as a type? the type looks good (bonus points for `unknown`) but it's making one more named type that doesn't seem to add any value over typing the `looksLikeApiInvocation` function.

If you want a generic interface to re-use, I'd rename to `AssertsApiInvocationIssue`, but even then fi we want that more generic functionality I'd say to pull it out into a utility module and have the type on the function.

Reply:
> thanks for your (out-of-band) help fixing the delcarations.

Does the current PR reflect this out-of-band effort yet?

Reply:
Yep. The typescript declarations reflect the current state of the code and make a lot of the API visible in an IDE.